examples/foreach/foreach.bal

```diff
@@ -33,7 +33,7 @@ public function main() {
             io:println("json array value: ", j);
         } else {
             // JSON is a union type for `()` or `null` | `int` | `float` | `decimal` | `string` | `json[]` | `map<json>`,
-            // `j` in else block if type cannot be inferred then it is type `anydata`.
+            // `j` will be in the else block if the type cannot be inferred. Type of `j` will be `anydata`.
```

Comment:
Not related to your change but is this the correct? 

Shouldn't we be saying "The `else` block is reached if `j` is neither `string`, `int` or `json[]`."

Also IINM, the type here will be `json`, not `anydata`.

Reply:
Changed with https://github.com/ballerina-platform/ballerina-lang/pull/14911/commits/d3c8c5a9532002ca935759535f1443f36accc8b6